app/main.py

```diff
@@ -38,7 +38,7 @@
 app.mount("/ui", StaticFiles(directory="app/static", html=True), name="static")
 
 # Set up PostgreSQL
-config = load_database_config("config.ini")
```

Comment:
This change will break the app if we want to run it locally rather than in docker